log-example/src/main/resources/log4j.properties

```diff
@@ -4,7 +4,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
 
 log4j.appender.http=io.phdata.pulse.log.HttpAppender
-log4j.appender.http.Address=http://edge2.valhalla.phdata.io:9015/v2/events/pulse-test-100
+log4j.appender.http.Address=http://edge2.valhalla.phdata.io:8214/v2/events/logging-pulse-test
```

Comment:
This is an example file, it shouldn't be updated as part of your commit. You can create your own version for your test.

Reply:
we will not add in next pull request.